backend-server/src/main/java/org/dice/ida/action/process/ActionMapper.java

```diff
@@ -0,0 +1,41 @@
+package org.dice.ida.action.process;
+
+import java.util.Map;
+
+import org.dice.ida.action.def.Action;
+import org.dice.ida.action.def.DefaultAction;
+import org.dice.ida.action.def.LoadDataAction;
+import org.dice.ida.action.def.SimpleTextAction;
+import org.dice.ida.model.Intent;
+public class ActionMapper {
+	
+	public static Action fetchActionInstance(String intentText, Map<String, String> paramData) {
+		Action action = null;
+		// return the instance for requested action
+		Intent intent = Intent.getForKey(intentText);
+		switch(intent) {
```

Comment:
Codacy found an issue: [A switch statement does not contain a break](https://app.codacy.com/gh/dice-group/IDA/pullRequest?prid=6037853)